mmeds/tools/qiime2.py

```diff
@@ -120,7 +120,54 @@ def demultiplex(self):
             # Reverse compliment the barcodes in the mapping file if using paired reads
             if 'paired_end' == self.doc.reads_type:
                 cmd = cmd[:3] + ['--p-rev-comp-mapping-barcodes '] + cmd[3:]
+            self.jobtext.append(' '.join(cmd))
+
         elif 'dual_barcodes':
+            self.add_path('pheniqs_config', '.json')
+            self.add_path(self.path / 'pheniqs_output', key='pheniqs_dir')
+            self.add_path(self.path / 'stripped_output', key='stripped_dir')
+
+            if not self.get_file('pheniqs_dir', True).is_dir():
+                self.get_file('pheniqs_dir', True).mkdir()
+
+            if not self.get_file('stripped_dir', True).is_dir():
+                self.get_file('stripped_dir', True).mkdir()
+
+            load = 'ml pheniqs'
+            self.module = load
+            self.jobtext.append(load)
+            cmd = [
+                'python make_pheniqs_config.py',
```

Comment:
no 'python' here or it'll try to find a script named `make_pheniqs_config.py`. Just run the script directly `make_pheniqs_config.py ...`